pkg/routeagent_driver/main.go

```diff
@@ -213,14 +226,10 @@ func uninstall(registry *event.Registry) {
 	}
 }
 
-func waitForNodeReady(k8sClientSet *kubernetes.Clientset) {
+func waitForNodeReady(localNode *corev1.Node) {
```

Comment:
Won't we be stuck in an infinite loop if node state != ready? B/C we only read node information once on line 115?

Reply:
yeah this code needs to read the localNode each time so it can't be passed in.